### PR TITLE
Fix typescript installation in ITs

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: its/plugin/tests/src/test/java/com/sonar/javascript/it/plugin/SonarLintTestCustomNodeJS.java [hunk 1]
```diff
@@ -82,8 +82,8 @@ public void should_use_set_nodejs() throws Exception {
       .addInputFile(inputFile)
       .putAllExtraProperties(properties).build();
     sonarlintEngine.analyze(configuration, issues::add, null, null);
+    sonarlintEngine.stop();
```
Comment (vilchik-elena) [inline its/plugin/tests/src/test/java/com/sonar/javascript/it/plugin/SonarLintTestCustomNodeJS.java]: why did you add this call?

File: Jenkinsfile [hunk 2]
```diff
@@ -45,7 +45,9 @@ pipeline {
             label 'linux'
           }
           steps {
-            runITs("ruling","LATEST_RELEASE")            
+            nodejs(configId: 'npm-artifactory', nodeJSInstallationName: 'NodeJS latest') {
```
Comment (vilchik-elena) [inline Jenkinsfile]: I don't understand why we need this

Comment (saberduck) [reply Jenkinsfile]: we need to have NodeJS available for ruling

File: its/plugin/tests/src/test/java/com/sonar/javascript/it/plugin/SonarLintTestCustomNodeJS.java (hunk 1 above)
Comment (saberduck) [reply its/plugin/tests/src/test/java/com/sonar/javascript/it/plugin/SonarLintTestCustomNodeJS.java]: because if sonarlintEngine adds something to the log after the call to analyze it can throw `ConcurrentModificationException` when we are checking the logs (you can't iterate the collection when it is modified). This is causing false positives on the QA

File: Jenkinsfile (hunk 2 above)
Comment (vilchik-elena) [reply Jenkinsfile]: how it was working before?)

Comment (saberduck) [reply Jenkinsfile]: it wasn't, sometimes we were lucky that we were on agent with NodeJS and sometimes it failed
